### ISSUE TITLE
More than three authors are not printed in bibliography

### ISSUE THREAD

**Issue by bernhard-thiele** [modelica/conference-templates#23]
I'm using the present conference-templates master on Ubuntu 18.04.5 LTS. If there are more than three authors, they won't be printed in the bibliography.

As an example consider following entry

```
Article{Stuermer:2007,
  Title                    = {Systematic Testing of Model-Based Code Generators},
  Author                   = {Stürmer, Ingo. and Conrad, Mirko and Dörr, Heiko and Pepper, Peter},
  Journal                  = {Software Engineering, IEEE Transactions on},
  Year                     = {2007},
  Number                   = {9},
  Pages                    = {622-634},
  Volume                   = {33},
  Doi                      = {10.1109/TSE.2007.70708},
}
```

Which results in following bibliography rendering:
![image](https://user-images.githubusercontent.com/4244614/114999599-4b4a8e80-9ea2-11eb-96f1-514d8bc8cb4b.png)

Is this intentional?

**Comment by sjoelund** [comment]
It is intentional to have a limit. 3 seems a bit low though. It can be changed by the biblatex option `maxbibnames`. I guess it should be decided and updated for the next conference, but feel free to increase it for your paper to something more appropriate (like 5?).

**Comment by bernhard-thiele** [comment]
I think the default should be even higher. Usually I'd like all authors to be printed. No doubt there are practical limits for some publications (https://www.nature.com/news/physics-paper-sets-record-with-more-than-5-000-authors-1.17567), but I haven't seen such monsters in Modelica papers, yet.

An example for a very long Modelica related title for which we bear some responsibility would render like
![image](https://user-images.githubusercontent.com/4244614/116219266-b5d5b700-a74b-11eb-8d3a-f37e2dedb331.png)

This is not very pretty, but even such a long entry doesn't use up practically prohibitively much space. Here we are close to 30 authors. I find it strange that the default for biber is only 3 authors. Not sure where to set the limit. As you suggested, it should be discussed and decided for the next conference.

**Comment by sjoelund** [comment]
https://github.com/modelica-association/conference-templates/blob/b1a17705f8ea892da43b7a29b3a806e550a30a8d/LaTeX/modelica.cls#L102 `maxbibnames=SOMENUMBER`. It's not that the entry is too long that is problematic but rather that you can't see the title as easily in my opinion.

**Comment by bernhard-thiele** [comment]
True. It's also simply not very pretty. The question is where to set the limit.

I think 3 is too small. 5 is already much better. Maybe even rather 7. Would be interesting to get opinions of the board.

**Comment by dietmarw** [comment]
It looks like there is a side-effect or rather a misconfiguration. Currently not only are three authors listed in the bibliography but also in the citation inside the key making those extremely long. For example in in the previous conference template the key was printed as:
 - (Kundur 1994)

but now it is: 
 - (Kundur, Balu, and Lauby 1994)

This disturbs the flow in the text considerably, especially when several citations are used and given the two-coumn mode which does not allow for much space anyway.

**Comment by sjoelund** [comment]
Two-column itself is problematic, yes. Of course the one for previous conferences is also wrong as it really should be et al. `maxcitenames=1` (or 2) should fix that issue (Kundur et al. 1994).

**Comment by dietmarw** [comment]
Do you think we could make the `maxcitenames=1` already now?

**Comment by sjoelund** [comment]
APA and MLA have it at 2 and you could set it at that. Chicago at 3. It's weird to say "Winkler et al" instead of "Winkler and Beutlich".
